src/test/rustdoc-gui/search-result-display.goml

```diff
@@ -15,26 +15,28 @@ assert-css: (".search-results div.desc", {"width": "570px"})
 // To do so we need to update the length of one of its `<option>`.
 size: (900, 900)
 
-// First we check the current width and position.
-assert-css: ("#crate-search", {"width": "222px"})
-compare-elements-position-near: (
-    "#crate-search",
-    "#search-settings .search-results-title",
-    {"y": 5},
-)
+// FIXME: Fix and re-enable these tests!
```

Comment:
So about the `goml` format. It's something I wrote to make it easy to write/read. You can see more information about it (with the full list of commands) [here](https://github.com/GuillaumeGomez/browser-UI-test/blob/master/goml-script.md#command-list).